ognjen/pay-pal/src/components/ThirdSection.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react"
+import Tab1 from "./Tab"
+import Tab2 from "./Tab"
+import Button from 'react-bootstrap/Button';    
+
+export default function ThirdSection ({children}) {
+    const [activeTab, setActiveTab] = useState('ForBuyers')
```

Comment:
Avoid hardcoding strings. Create object literal schema.

```suggestion
    const activeTabSchema = {
        FOR_SELLERS: "ForSellers",
        FOR_BUYERS: "ForBuyers"
    }

    const [activeTab, setActiveTab] = useState(activeTabSchema.FOR_SELLERS)
```